windows-oci-context/README.md

```diff
@@ -0,0 +1,37 @@
+# Using an OCI context
+
+The sample below builds a windows image based on an OCI context.
+
+> These are `powershell` commands. 
+
+## Bundle up the context into an OCI Artifact
+
+The context has a simple `Dockerfile` and a `hello.txt`
+Use oras to create the context and ensure you are in `context` directory. 
+
+```shell
+$REGISTRY="myregistry"
+$REGISTRY_LOGIN_SERVER="myregistry.azurecr.io"
+cd context
+az acr login -n $REGISTRY
+oras push $REGISTRY_LOGIN_SERVER/windows/oci-context:latest .
+```
+
+## Create an ACR to use the context
+
+Create a base image trigger enabled task with the new context and ensure that you specify the required `--platform`. 
+
+
+```powershell
+$REGISTRY="myregistry"
+$REGISTRY_LOGIN_SERVER="myregistry.azurecr.io"
+az acr task create `
+  --registry $REGISTRY `
+  --name ocicontextbuild `
+  --platform Windows/amd64 `
+  --image "windowsimage:{{.Run.ID}}" `
+  --context "oci://$REGISTRY_LOGIN_SERVER/windows/oci-context:latest" `
+  --file Dockerfile `
+  --commit-trigger-enabled false `
+  --base-image-trigger-enabled true
+```
```

Comment:
can you add a step like "az acr task run --name ocicontextbuild"?

Reply:
Update and added outputs as well.